src/types/game.d.ts

```diff
@@ -4,6 +4,11 @@ declare type Option = {
   value: string;
   x: number;
   y: number;
+  velh: number;
+  velv: number;
+  acelh: number;
+  acelv: number;
```

Comment:
velh, velv, acelh and acelv aren't very clear names for these props.

```suggestion
  speedH: number;
  speedV: number;
  accelerationH: number;
  accelerationV: number;
```

The suggestion I left has better names for these properties, you should still change them in other areas of the code.

Reply:
I agree, i will change that!